CONTRIBUTING.md

```diff
@@ -0,0 +1,196 @@
+**A:** [Management of python dependencies](docs/python-dependencies.md) has a detailed explanation and useful resources.
+
+**Q:** I see and `api` folder full of text files, what is it?
+**A:** Look at [APIs development guidelines](api/README.md) it will point you in the right direction.
+
+**Q:** Do you need to commit something (maybe work in progress) and it makes no sense to have the CI run it?
+**A:** Add `[ci skip]` in any part of your commit message. The CI will not run get started.
+
+**Q:** I don't have a good understanding of the aims of the project. Where can I read more?
+**A:** [D1.1 Technical Specs. & Technology Evaluation](https://osparc-docs.readthedocs.io/en/latest/) is extensive, but explains the decision making behind most of the code. Please consider reading it. Afterwords you can better grasp the [Computational backend](docs/comp-services.md) description.
+
+**Q:** Can I have some sort of introduction video about the project?
+**A:** Have a look at the following videos, targeted for wider audiences: [What is the SPARC program?](https://www.youtube.com/watch?v=IrZS_IHgrcE&feature=emb_title), [Ideas Lab Introduction to o²S²PARC](https://www.youtube.com/watch?time_continue=11&v=Q96jyFgSFHc&feature=emb_title) and [SPARC Webinar: Introduction to o²S²PARC](https://www.youtube.com/watch?v=vrdVealYydE&feature=emb_title)
+
+---
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
```

Comment:
```suggestion
* Saying things like only backend guys are real men and front end is for pussies is not acceptable.
* Calling your colleagues "lazy spaniards", even if it's true. They just like to sleep "siesta" and go back to work with renovated energies.

```

Reply:
AHAH this suggestion is amazing

Reply:
I will megere it if you give it 10 👍

Reply:
what kind of democracy is that? 50% of :+1: should be enough